### DISCUSSION TITLE
How to close MongoHook

### DISCUSSION THREAD

**Discussion by cphpitts-krs** [apache/airflow#30535] [Q&A]
I noticed that in the update to 3.1.1, the MongoHook lose the close_conn() method. Is there another way to Mongo connections now?

**Comment by potiuk** [comment]
can you point to the sources of loosing the close_conn? What the release notes tell you?

**Reply by cphpitts-krs** [reply]
Shown being removed in the latest commit
https://github.com/apache/airflow/commit/6af796f4592429d9ddfcb7a1468e5dd7ea8ee1af#diff-c433c6cf79c2912d479dd5a1e3b68c8f5a9d5925ec5ad0f3e27a9936a7bc8986L96

Also no longer listed in the documentation. 
https://airflow.apache.org/docs/apache-airflow-providers-mongo/stable/_api/airflow/providers/mongo/hooks/mongo/index.html

Don't see any mention in release notes

**Reply by hussein-awala** [reply]
This is just one commit in the PR of 18 commits, you need to check the whole PR changes:
```diff
    def __exit__(
        self,
        exc_type: type[BaseException] | None,
        exc_val: BaseException | None,
        exc_tb: TracebackType | None,
    ) -> None:
        if self.client is not None:
-          self.close_conn()
+          self.client.close()
+          self.client = None
```

**Comment by potiuk** [comment]
What looking at the change introduced there tell you ? https://github.com/apache/airflow/pull/28675/files
